doc/dependency_visualization.md

```diff
@@ -5,6 +5,12 @@
 You can use stack to visualize the dependencies between your packages and
 optionally also external dependencies.
 
+First, you need to install Graphviz:
+
+```
+$ brew install Graphviz
```

Comment:
This seems to be Mac-specific, and that should be called out.

Reply:
I have fixed it